### ISSUE TITLE
2025-01-25, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#62]
# Today's Arxiv Papers

(1) [CRPO: Confidence-Reward Driven Preference Optimization for Machine Translation](http://arxiv.org/abs/2501.13927v1), 2025-01-23

(2) [The Breeze 2 Herd of Models: Traditional Chinese LLMs Based on Llama with Vision-Aware and Function-Calling Capabilities](http://arxiv.org/abs/2501.13921v1), 2025-01-23

(3) [Analysis of Indic Language Capabilities in LLMs](http://arxiv.org/abs/2501.13912v1), 2025-01-23

(4) [Privacy-Preserving Personalized Federated Prompt Learning for Multimodal Large Language Models](http://arxiv.org/abs/2501.13904v1), 2025-01-23

(5) [Exploring Finetuned Audio-LLM on Heart Murmur Features](http://arxiv.org/abs/2501.13884v1), 2025-01-23

(6) [The machine learning platform for developers of large systems](http://arxiv.org/abs/2501.13881v1), 2025-01-23

(7) [A RAG-Based Institutional Assistant](http://arxiv.org/abs/2501.13880v1), 2025-01-23

(8) [On the Reasoning Capacity of AI Models and How to Quantify It](http://arxiv.org/abs/2501.13833v1), 2025-01-23

(9) [Predicting Compact Phrasal Rewrites with Large Language Models for ASR Post Editing](http://arxiv.org/abs/2501.13831v1), 2025-01-23

(10) [Hallucinations Can Improve Large Language Models in Drug Discovery](http://arxiv.org/abs/2501.13824v1), 2025-01-23

(11) [Large Language Model driven Policy Exploration for Recommender Systems](http://arxiv.org/abs/2501.13816v1), 2025-01-23

(12) [Enhancing LLMs for Governance with Human Oversight: Evaluating and Aligning LLMs on Expert Classification of Climate Misinformation for Detecting False or Misleading Claims about Climate Change](http://arxiv.org/abs/2501.13802v1), 2025-01-23

(13) [Not Every AI Problem is a Data Problem: We Should Be Intentional About Data Scaling](http://arxiv.org/abs/2501.13779v1), 2025-01-23

(14) [Explainable XR: Understanding User Behaviors of XR Environments using LLM-assisted Analytics Framework](http://arxiv.org/abs/2501.13778v1), 2025-01-23

(15) [Do Large Language Models Truly Understand Geometric Structures?](http://arxiv.org/abs/2501.13773v1), 2025-01-23

(16) [Tune In, Act Up: Exploring the Impact of Audio Modality-Specific Edits on Large Audio Language Models in Jailbreak](http://arxiv.org/abs/2501.13772v1), 2025-01-23

(17) [UGMathBench: A Diverse and Dynamic Benchmark for Undergraduate-Level Mathematical Reasoning with Large Language Models](http://arxiv.org/abs/2501.13766v1), 2025-01-23

(18) [EICopilot: Search and Explore Enterprise Information over Large-scale Knowledge Graphs with LLM-driven Agents](http://arxiv.org/abs/2501.13746v1), 2025-01-23

(19) [GPT-HTree: A Decision Tree Framework Integrating Hierarchical Clustering and Large Language Models for Explainable Classification](http://arxiv.org/abs/2501.13743v1), 2025-01-23

(20) [An Empirical Study of Retrieval-Augmented Code Generation: Challenges and Opportunities](http://arxiv.org/abs/2501.13742v1), 2025-01-23

(21) [Pseudocode-Injection Magic: Enabling LLMs to Tackle Graph Computational Tasks](http://arxiv.org/abs/2501.13731v1), 2025-01-23

(22) [RPO: Retrieval Preference Optimization for Robust Retrieval-Augmented Generation](http://arxiv.org/abs/2501.13726v1), 2025-01-23

(23) [Musical ethnocentrism in Large Language Models](http://arxiv.org/abs/2501.13720v1), 2025-01-23

(24) [EventVL: Understand Event Streams via Multimodal Large Language Model](http://arxiv.org/abs/2501.13707v1), 2025-01-23

(25) [DI-BENCH: Benchmarking Large Language Models on Dependency Inference with Testable Repositories at Scale](http://arxiv.org/abs/2501.13699v1), 2025-01-23

(26) [Question Answering on Patient Medical Records with Private Fine-Tuned LLMs](http://arxiv.org/abs/2501.13687v1), 2025-01-23

(27) [HumorReject: Decoupling LLM Safety from Refusal Prefix via A Little Humor](http://arxiv.org/abs/2501.13677v1), 2025-01-23

(28) [How to Complete Domain Tuning while Keeping General Ability in LLM: Adaptive Layer-wise and Element-wise Regularization](http://arxiv.org/abs/2501.13669v1), 2025-01-23

(29) [LVPruning: An Effective yet Simple Language-Guided Vision Token Pruning Approach for Multi-modal Large Language Models](http://arxiv.org/abs/2501.13652v1), 2025-01-23

(30) [Sigma: Differential Rescaling of Query, Key and Value for Efficient Language Models](http://arxiv.org/abs/2501.13629v1), 2025-01-23

(31) [FedPref: Federated Learning Across Heterogeneous Multi-objective Preferences](http://dx.doi.org/10.1145/3708984), 2025-01-23

(32) [Text-to-SQL based on Large Language Models and Database Keyword Search](http://arxiv.org/abs/2501.13594v1), 2025-01-23

(33) [Improving Contextual Faithfulness of Large Language Models via Retrieval Heads-Induced Optimization](http://arxiv.org/abs/2501.13573v1), 2025-01-23

(34) [LLMs Can Plan Only If We Tell Them](http://arxiv.org/abs/2501.13545v1), 2025-01-23

(35) [ReasVQA: Advancing VideoQA with Imperfect Reasoning Process](http://arxiv.org/abs/2501.13536v1), 2025-01-23

(36) [RECALL: Library-Like Behavior In Language Models is Enhanced by Self-Referencing Causal Cycles](http://arxiv.org/abs/2501.13491v1), 2025-01-23

(37) [Adaptive Testing for LLM-Based Applications: A Diversity-based Approach](http://arxiv.org/abs/2501.13480v1), 2025-01-23

(38) [Streaming Video Understanding and Multi-round Interaction with Memory-enhanced Knowledge](http://arxiv.org/abs/2501.13468v1), 2025-01-23

(39) [Spurious Forgetting in Continual Learning of Language Models](http://arxiv.org/abs/2501.13453v1), 2025-01-23

(40) [MultiDreamer3D: Multi-concept 3D Customization with Concept-Aware Diffusion Guidance](http://arxiv.org/abs/2501.13449v1), 2025-01-23

(41) [Softplus Attention with Re-weighting Boosts Length Extrapolation in Large Language Models](http://arxiv.org/abs/2501.13428v1), 2025-01-23

(42) [VulnBot: Autonomous Penetration Testing for A Multi-Agent Collaborative Framework](http://arxiv.org/abs/2501.13411v1), 2025-01-23

(43) [Can Large Language Models Understand Preferences in Personalized Recommendation?](http://arxiv.org/abs/2501.13391v1), 2025-01-23

(44) [Do as We Do, Not as You Think: the Conformity of Large Language Models](http://arxiv.org/abs/2501.13381v1), 2025-01-23

(45) [50 Shades of Deceptive Patterns: A Unified Taxonomy, Multimodal Detection, and Security Implications](http://arxiv.org/abs/2501.13351v1), 2025-01-23

(46) [Full-Stack Optimized Large Language Models for Lifelong Sequential Behavior Comprehension in Recommendation](http://arxiv.org/abs/2501.13344v1), 2025-01-23

(47) [Multi-aspect Knowledge Distillation with Large Language Model](http://arxiv.org/abs/2501.13341v1), 2025-01-23

(48) [AgentRec: Agent Recommendation Using Sentence Embeddings Aligned to Human Feedback](http://arxiv.org/abs/2501.13333v1), 2025-01-23

(49) [Qrazor: Reliable and effortless 4-bit llm quantization by significant data razoring](http://arxiv.org/abs/2501.13331v1), 2025-01-23

(50) [SplitLLM: Hierarchical Split Learning for Large Language Model over Wireless Network](http://arxiv.org/abs/2501.13318v1), 2025-01-23

(51) [Representing Visualization Insights as a Dense Insight Network](http://arxiv.org/abs/2501.13309v1), 2025-01-23

(52) [OSUM: Advancing Open Speech Understanding Models with Limited Resources in Academia](http://arxiv.org/abs/2501.13306v1), 2025-01-23

(53) [Watching the AI Watchdogs: A Fairness and Robustness Analysis of AI Safety Moderation Classifiers](http://arxiv.org/abs/2501.13302v1), 2025-01-23

(54) [Hypothesis Generation for Materials Discovery and Design Using Goal-Driven and Constraint-Guided LLM Agents](http://arxiv.org/abs/2501.13299v1), 2025-01-23

(55) [RAMQA: A Unified Framework for Retrieval-Augmented Multi-Modal Question Answering](http://arxiv.org/abs/2501.13297v1), 2025-01-23

(56) [Toyteller: AI-powered Visual Storytelling Through Toy-Playing with Character Symbols](http://dx.doi.org/10.1145/3706598.3713435), 2025-01-23
